src/sidebar/components/MentionSuggestionsPopover.tsx

```diff
@@ -2,11 +2,61 @@ import { Popover } from '@hypothesis/frontend-shared';
 import type { PopoverProps } from '@hypothesis/frontend-shared/lib/components/feedback/Popover';
 import classnames from 'classnames';
 
-export type UserItem = {
-  username: string;
-  displayName: string | null;
+import { isThirdPartyUser } from '../helpers/account-id';
+import type { UserItem } from '../helpers/mention-suggestions';
+
+type SuggestionItemProps = {
+  user: UserItem;
+  usersListboxId: string;
+  highlighted: boolean;
+  isThirdParty: boolean;
```

Comment:
At this level, the configuration that we want to pass to the suggestion item is whether to show usernames or not. The `SuggestionItem` component doesn't need to know the details of the policy used to determine this.